prog/vm/nexus.rb

```diff
@@ -41,9 +41,7 @@ def self.assemble(public_key, project_id, name: nil, size: "m5a.2x",
       # - If the user provided neither nic_id nor private_subnet_id, that's OK, we create both.
       nic = nil
       subnet = nil
-      if nic_id
-        nic = Nic[nic_id]
-        raise("Given nic doesn't exist with the id #{nic_id}") unless nic
```

Comment:
I think we should keep existence check here too. For example, operator typed `nic_id` wrong and expects to use existing nic. But code will generate new nic. Same for `private_subnet_id` too

Reply:
I missed the operator error part, you're right. Added back.